Tests/SwiftStdlibTests/StringExtensionsTests.swift

```diff
@@ -312,6 +312,11 @@ final class StringExtensionsTests: XCTestCase {
         XCTAssertEqual(helloWorld.localized(comment: "comment"), NSLocalizedString(helloWorld, comment: "comment"))
     }
 
+    func testFormatLocalized() {
+        XCTAssertEqual("%d Swift %d Objective-C".formatLocalized(1, 2), NSLocalizedString("1 Swift 2 Objective-C", comment: ""))
+        XCTAssertEqual("%d Swift %d Objective-C".formatLocalized(comment: "comment", 1, 2), NSLocalizedString("1 Swift 2 Objective-C", comment: "comment"))
```

Comment:
You want to test the end result, not the internal implementation of the function.
```suggestion
        XCTAssertEqual("%d Swift %d Objective-C".formatLocalized(1, 2), "1 Swift 2 Objective-C")
        XCTAssertEqual("%d Swift %d Objective-C".formatLocalized(comment: "comment", 1, 2), "1 Swift 2 Objective-C")
```